commands/user.go

```diff
@@ -402,3 +427,84 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 
 	return nil
 }
+
+func generateTokenForAUserCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) != 2 {
+		return errors.New("expected at least two arguments. See help text for details")
+	}
+
+	userArg := args[0]
+	user := getUserFromUserArg(c, userArg)
+	if user == nil {
+		return errors.Errorf("could not retrieve user information of %q", userArg)
+	}
+
+	token, res := c.CreateUserAccessToken(user.Id, args[1])
+	if res.Error != nil {
+		return errors.Errorf("could not create token for %q: %s", userArg, res.Error.Error())
+	}
+	printer.Print(fmt.Sprintf("%s : %s", token.Token, token.Description))
+
+	return nil
+}
+
+func listTokensOfAUserCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return errors.New("expected at least one argument. See help text for details")
+	}
+
+	page, err := command.Flags().GetInt("page")
+	if err != nil {
+		return err
+	}
+
+	perPage, err := command.Flags().GetInt("per-page")
+	if err != nil {
+		return err
+	}
+
+	showAll, err := command.Flags().GetBool("all")
+	if err != nil {
+		return err
+	}
+
+	active, err := command.Flags().GetBool("active")
+	if err != nil {
+		return err
+	}
+
+	inactive, err := command.Flags().GetBool("inactive")
+	if err != nil {
+		return err
+	}
+
+	if showAll {
+		page = 0
```

Comment:
Should we set `perPage` to some high number like `9999` in this case to fetch as many tokens as possible?

Reply:
Added. 👍